### PR TITLE
feature/FOUR-18630

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
FOUR-18498: FE: Needs to collpase/descollapse according the flag saved

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-18630

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

File: resources/js/processes-catalogue/components/ProcessesCatalogue.vue [hunk 1]
```diff
@@ -136,9 +139,28 @@ export default {
     }
   },
   methods: {
+    getUserConfiguration() {
+      ProcessMaker.apiClient
+        .get(this.urlConfiguration)
+        .then((response) => {
+          this.userConfiguration = JSON.parse(response.data.ui_configuration);
+          this.showMenu = this.userConfiguration.launchpad.isMenuCollapse;
+        });
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/ProcessesCatalogue.vue:148]: add a catch to get errors from the API

File: resources/js/processes-catalogue/components/ProcessesCatalogue.vue [hunk 2]
```diff
@@ -136,9 +139,28 @@ export default {
...
+      this.updateUserConfiguration();
+    },
+    updateUserConfiguration() {
+      this.userConfiguration.launchpad.isMenuCollapse = this.showMenu;
+      ProcessMaker.apiClient
+        .put(
+          this.urlConfiguration,
+          {
+            ui_configuration: this.userConfiguration,
+          },
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/ProcessesCatalogue.vue:162]: add .catch when the API give an error

**Review by pmPaulis — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/ProcessesCatalogue.vue (hunk 1 above)
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessesCatalogue.vue:148]: is there a possibility to get the entire configuration from the Controller?
So we avoid making several Call to API

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-4317444e89.engk8s.processmaker.net

**Pull request merged**